rpm/SOURCES/cntlm-user

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# cNTLM per user launcher script
+USERNAME="$(id -un)"
+HOME="$(getent passwd "$(id -u)" | cut -d: -f6)"
```

Comment:
Aren't these variables already set? It seems surprising you have to set them manually.

Reply:
Will double check here with a printenv as I'm honestly not 100% certain.

systemd uses a non login shell for running services therefore not all variables that one would expect are set.

Reply:
Above are confirmed redundant, however there are no XDG_* variables set as this is a non-login shell.

This is the result of printenv at the top of my launcher script when invoked as a systemd system scoped unit but with the user parameter set to ec2-user
```
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal systemd[1]: Started cNTLM authentication proxy - per user parameterized service.
░░ Subject: A start job for unit cntlm@ec2-user.service has finished successfully
░░ Defined-By: systemd
░░ Support: https://access.redhat.com/support
░░ 
░░ A start job for unit cntlm@ec2-user.service has finished successfully.
░░ 
░░ The job identifier is 3621.
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal cntlm-user[26222]: SHELL=/bin/bash
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal cntlm-user[26222]: PWD=/
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal cntlm-user[26222]: LOGNAME=ec2-user
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal cntlm-user[26222]: SYSTEMD_EXEC_PID=26221
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal cntlm-user[26222]: HOME=/home/ec2-user
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal cntlm-user[26222]: LANG=en_US.UTF-8
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal cntlm-user[26222]: INVOCATION_ID=1549134fdb3b4048ad57ae5864ec06fc
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal cntlm-user[26222]: USER=ec2-user
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal cntlm-user[26222]: SHLVL=1
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal cntlm-user[26222]: JOURNAL_STREAM=8:53774
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal cntlm-user[26222]: PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin
Mar 06 23:30:52 ip-10-0-15-54.us-east-2.compute.internal cntlm-user[26222]: _=/usr/bin/printenv
````